### ISSUE TITLE
Re-rendering works not correctly?

### ISSUE THREAD

**Issue by LiguidCool** [leonidas/transparency#151]
https://jsfiddle.net/es8behkn/4/
I want, load many JSON files, but I can't render them. What do I do wrong?

**Comment by woubuc** [comment]
When you call `.render` on an element, it goes back to its original state as defined in the HTML. Try wrapping them in separate elements, like so: https://jsfiddle.net/edu1gp3a/

Alternatively, you could first do all AJAX requests, and then combine the data in 1 object and render it all at once.

**Comment by LiguidCool** [comment]
Yep, that works. But if i use `$('body').render(data);` (or `<html>`? I want rerender whole html file ...
May be wrappers can help, but wrappers in wrappers etc ... Not good.

**Comment by LiguidCool** [comment]
https://jsfiddle.net/8uau1qo7/1/
I can use async.js, but i think that also bad practice.
